Automattic/wp-calypso
# Happy blocks: Update WPCom nav bar for happy blocks

## Proposed Changes

The main idea of this PR is to **not** show the WPCom to logged-in users. For those, Calypso bar is the only one visible.
Besides that, it also does:
- Visually matches the logged out [WPCom](https://wordpress.com/) nav bar.
- For logged-in users, only load header.scss which contains the `happy blocks` header styles.
- For logged-out users, it loads both, header.scss and view.scss (nav bar styles).
- CSS clean up.

Before: 
<img width="2041" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/d1b8f619-6543-47b3-af2f-2e12659661fb">


After:
<img width="2046" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/eded5e81-e90b-4389-a431-e66b39fd2650">


## Testing Instructions

- Pull and run this branch
- Navigate to `cd apps/happy-blocks` and `run yarn dev --sync`
- Sandbox wordpress.com/support and visit the site

For logged-in users: 
 - The header should not be present.
 - Check on the network tab if view.scss for the header is loaded, **It should not**. The only one visible is the footer one. 

For logged-out users:
 - The header should be present.
 - Check on the network tab if view.scss for the header is loaded, **It should**. 


*** No changes on mobile
